### PR TITLE
Added the logout button to profiles page

### LINKED ISSUE #36: Allow users to log out
**Is your feature request related to a problem? Please describe.**
The user can currently log in but cannot log out

**Describe the solution you'd like**
Allow the user to log out from their profile page

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by JC4github**
Resolves #36

### CODE REVIEW

File: app/pages/profile/page.tsx [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
 export default function Profile() {
-    return (
-        <div>
-            <div className="flex flex-col justify-center items-center text-3xl">
-                <div className="font-bold w-1/5 text-center py-[200px]">
-                    Welcome to your profile page.
-                </div>
-            </div>
+  return (
+    <div>
```
Comment (jaskfla) [inline app/pages/profile/page.tsx]: Surely we can remove most of these `div`s?

File: app/pages/profile/page.tsx [hunk 2]
```diff
@@ -1,11 +1,16 @@
...
+      <div className="flex flex-col justify-center items-center text-3xl">
+        <div className="font-bold w-1/5 text-center py-[200px]">
+          Welcome to your profile page.
```
Comment (jaskfla) [inline app/pages/profile/page.tsx]: Does the text belong in a `<p>` tag?

**Review by jaskfla — COMMENTED** [review]
Looks pretty all right to me, but a bunch of redundant `div`s? Not sure if that’s just a consequence of the framework.

File: app/pages/profile/page.tsx (hunk 1 above)
Comment (JC4github) [reply app/pages/profile/page.tsx]: I have removed the top-level div

File: app/pages/profile/page.tsx (hunk 2 above)
Comment (JC4github) [reply app/pages/profile/page.tsx]: I have added a <p> tag around the text

**Review by hwan513 — APPROVED** [review]
I think this works as a temporary solution. 
Later on, we could consider moving the logout button into a nav dropdown, which most people would be familiar with.

**Pull request merged**